### DISCUSSION TITLE
🛫 Version 1.2.2

### DISCUSSION THREAD

**Discussion by valnoxy** [valnoxy/t480-oc#54] [Announcements]
## Changes
This pull request combines all the different macOS configurations into a single configuration (except Heliport). This allows an easy kext patching depending on the kernel version.

The following kexts was added:
   - USBMap (USB mapping)
   - VoodooI2C (Touchscreen Support)
   - VoodooI2CHID (Touchscreen Support)

The following base / kexts was updated:
   - **Airportitlwm**: (Ventura: 2.2.0 DEBUG, Monterey & Big Sur: 2.1.0)
   - **AppleALC**: 1.7.4
   - **HibernationFixup**: 1.4.6
   - **IntelBluetoothFirmware**: 2.2.0
   - **Lilu**: 1.6.2
   - **NVMeFix**: 1.1.0
   - **VirtualSMC (with plugins)**: 1.3.0
   - **VoodooPC2Controller**: 2.2.9
   - **WhateverGreen**: 1.6.1
   - **OpenCore**: 0.8.5

The following config changes was applied:
   - **Added Thinkpad ACPI / Kernel patches**
   - **Removed 'LegacyEnable'**

The following ACPI changes was applied:
   - **Added SSDT-KBTA.aml with source code (ThinkPadAssistant)**

## Issues assigned to this release
   - #26
   - #36 
   - #18
   - #29
   - #27

<hr /><em>This discussion was created from the release <a href='https://github.com/valnoxy/t480-oc/releases/tag/v1.2.2'>🛫 Version 1.2.2 </a>.</em>

**Comment by askwakhid** [comment]
Hi,

Thank you for the update.
I fork your repo.
https://github.com/askwakhid/ThinkPad-T480s-macOS-OpenCore

I think your EFI works good for my T480s with macoS Ventura 13.0.1.
I make 1 minor changes for make my USB-C to HDMI adaptor works.

Thank you.

**Reply by enosmarley** [reply]
How is it going now? Also, did you update opencore to the latest version and the kext or you left them like that?
